index.js

```diff
@@ -45,7 +45,7 @@ function transformFilename(file) {
 	file.revOrigBase = file.base;
 	file.revHash = revHash(file.contents);
 
-	var extIndex = file.path.indexOf('.');
+	var extIndex = file.path.indexOf('.', file.path.lastIndexOf('/'));
```

Comment:
Just to be safe I think we should operate on the `path.basename()` here instead and at the end use https://github.com/sindresorhus/modify-filename. That way we let Node handle all the path logic which I easy to do incorrectly.